cores/cv32e40p/user_manual/source/intro.rst

```diff
@@ -110,8 +110,18 @@ CV32E40P currently supports the following features according to the RISC-V Privi
 * Hardware Performance Counters as described in :ref:`performance-counters` based on ``NUM_MHPMCOUNTERS`` parameter
 * Trap handling supporting direct mode or vectored mode as described at :ref:`exceptions-interrupts`
 
+
+Synthesis guidelines
+--------------------
+
+The CV32E40P core has been taped-out in several nodes (from 130nm to 22nm)
+and tested on silicon many times.
+
+The ``constraints/cv32e40p_core.sdc`` file provides an example of synthesis
+constraints.
+
```

Comment:
Maybe add a remark about which files to synthesize (all in rtl/* excluding one of the two register file alternatives and excluding the pmp file, all in rtl/include/*, plus user supplied clock gate cell)